common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -560,6 +573,8 @@ function(VideoPlayer, i18n, moment, _) {
         _setConfigurations(this);
 
         if (!(_parseYouTubeIDs(this))) {
+            // Extract hls video sources
```

Comment:
This part code is removed in a follow up [PR](https://github.com/edx/edx-platform/pull/14754/files#diff-da1b19cc5f03a6f278a0c5a14616f7a5L576)